features/schemas/cve_fix_plan.json

```diff
@@ -0,0 +1,91 @@
+{
+  "type": "object",
+  "required": [ "cves_data" ],
+  "properties": {
+    "cves_data": {
+      "type": "object",
+      "properties": {
+        "expected_status": {
+          "type": "string"
+        },
+        "cves": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "title": {
+                "type": "string"
+              },
+              "expected_status": {
+                "type": "string"
+              },
+              "plan": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "operation": {
+                      "type": "string"
+                    },
+                    "data": {
+                      "type": "object",
+                      "patternProperties": {
+                        "^.*$": {
+                          "anyOf": [
+                            {"type": "string"},
+                            {"type": "array", "items": {"type": "string"}}
+                          ]
+                        }
+                      },
+                      "additionalProperties": false
+                    },
```

Comment:
We could make this more complete by specifying every possible data object using `anyOf`, but not sure if it's worth it. This is good for now.

Reply:
I am okay with either approach. What do you think @renanrodrigo ?

Reply:
Maybe a future iteration could cover the specification of the objects?

Reply:
Works for me +1